cmd/acb/commands/exec/exec.go

```diff
@@ -201,55 +200,67 @@ var Command = cli.Command{
 			}
 		}
 
+		// Add all creds provided by the user in the --credential flag
+		credentials, err := graph.CreateRegistryCredentialFromList(creds)
+		if err != nil {
+			return err
+		}
+
 		versionInUse := graph.FindVersion(template.GetData())
-		shouldIncludeAlias := versionInUse == "" || versionInUse >= "v1.1.0"
+		shouldIncludeAlias := versionInUse >= "v1.1.0"
 
+		var alias *graph.Alias
 		var task *graph.Task
 		if shouldIncludeAlias {
 			log.Printf("Alias support enabled for version >= 1.1.0, please see https://aka.ms/acr/tasks/task-aliases for more information.")
-			// Generate the base task file without resolving environment variables.
-			task, err = graph.NewTaskFromBytes(template.GetData(), true)
-			if err != nil {
-				return err
-			}
+			// separate alias and remaining data
+			aliasData, remainingData := graph.SeparateAliasFromRest(template.GetData())
+			// render alias data
+			values := templating.GetTaskRenderObject(renderOpts)
```

Comment:
>GetTaskRenderObject [](start = 24, length = 19)

GetTaskRenderObject only returns built-in values. I think alias also support custom values like

myAlias: {{.Values.MyValue}}?